2Homework25.02.22/Trie/Trie.Tests/TestTrie.cs

```diff
@@ -0,0 +1,75 @@
+using NUnit.Framework;
+
+namespace Trie.Tests;
+
+public class Tests
+{
+
+    [Test]
+    public void TestTrieAddAndSize()
+    {
+        Trie trie = new Trie();
```

Comment:
```suggestion
        var trie = new Trie();
```
А ещё это можно было в SetUp вынести, чтобы не писать в каждом тесте